rcljava/src/test/java/org/ros2/rcljava/action/ActionServerTest.java

```diff
@@ -41,14 +41,13 @@
     public test_msgs.action.Fibonacci_Goal goal;
     public GoalResponse handleGoal(test_msgs.action.Fibonacci.SendGoalRequest goal) {
       this.goal = goal.getGoal();
-      return GoalResponse.ACCEPT;
+      return GoalResponse.ACCEPT_AND_EXECUTE;
     }
   }
 
   class MockCancelCallback implements CancelCallback<test_msgs.action.Fibonacci> {
     public ActionServerGoalHandle<test_msgs.action.Fibonacci> goalHandle;
     public CancelResponse handleCancel(ActionServerGoalHandle<test_msgs.action.Fibonacci> goalHandle) {
-      this.goalHandle = goalHandle;
```

Comment:
I thought this was needed for the test (e.g. at L229).

Reply:
Ups, that was unintentionally pushed.
Actually, I've reverted the change locally and I forgot to commit it.

See https://github.com/osrf/ros2_java/pull/59/commits/085f579064958cc2ca75a066a742240e7d15ad00.